test_configs/sockets/quick.test

```diff
@@ -51,7 +51,7 @@
 		FT_COMP_QUEUE
 	],
 	eq_wait_obj: [
-		FI_WAIT_NONE
+		FI_WAIT_NONE,
```

Comment:
The parser is not set to strict JSON parsing. Hence it allows "lists" without commas.